src/main/java/com/maxmind/minfraud/response/FactorsResponse.java

```diff
@@ -70,11 +78,21 @@ public FactorsResponse(
         Integer queriesRemaining,
         Double riskScore,
         ShippingAddress shippingAddress,
+        List<RiskScoreReason> riskScoreReasons,
```

Comment:
This is a deprecated constructor that only exists for compatibility. You shouldn't be adding new parameters.

We will also need to add another deprecated constructor that matches the main constructor before `RiskScoreReasons`  was added.